### ISSUE TITLE
[owc + rollup build] Website not loaded on chrome desktop incognito & chrome mobile

### ISSUE THREAD

**Issue by apennamen** [vaadin/vaadin-router#451]
Hello,

## TLDR;
I used the default starter kit from Open Web Component, and added vaadin router 1.7.1
Once built, the website will load normally on chrome desktop (80.0.3987.163 (Build officiel) (64 bits)), but not in incognito mode, and neither on chrome mobile (80.0.3987.162).

## Steps to reproduce
[I made a repo to help reproduce the problem](https://github.com/apennamen/vaadin-router-bug)

1. Init repo
`npm init @open-wc`
Choose rollup

2. Add router as dependency
`yarn add @vaadin/router`

3. Modify index.html to add a "main" html tag, and use the following (don't mind the name of the view):
```javascript
import { Router } from '@vaadin/router';
import { WiredCalendarBug } from './WiredCalendarBug.js';

customElements.define('wired-calendar-bug', WiredCalendarBug);

function initRouter() {
    const router = new Router(document.querySelector('main'));
    router.setRoutes([
      {
        path: '/',
        component: 'wired-calendar-bug',
      },
    ]);
}
  
window.addEventListener('load', () => {
    console.log('lol');
    initRouter();
});
```

4. Build and launch:
```
yarn build
yarn start:build
```

5. Result:
![image](https://user-images.githubusercontent.com/7101875/79147496-4f8e2880-7dc4-11ea-93c8-775924cd1059.png)

## Why do I suspet vaadin router ?
Without vaadin router, the app works in both incognito mode and normal mode. Same for the mobile version.
There is no error in the console, however on the performance panel I can see that vaadin-router is called.
![image](https://user-images.githubusercontent.com/7101875/79147871-f1157a00-7dc4-11ea-9ec2-bbb292ff9888.png)

**Comment by apennamen** [comment]
OK I found the root cause 5 minutes after posting this issue...

It's because of this:
```javascript
window.addEventListener('load', () => {
    initRouter();
});
```

In incognito mode and on mobile, this event is not detected (I don't know why...)
Problem is, this indication is in all the docs I found on vaadin router.

I "fixed" this by replacing directly by a call, without the addEventListener (as if it was an _iife_) :
```javascript
initRouter();
```
